evap/staff/templates/staff_semester_view.html

```diff
@@ -31,6 +31,7 @@ <h3 class="m-0 me-1">{{ semester.name }}</h3>
                         >
                             {% translate 'Make active' %}
                         </button>
+				<span class="ms-2 fas fa-circle-info" data-bs-toggle="tooltip" data-bs-placement="right" title="{% translate 'The active semester is the semester that is initially opened for managers, reviewers and grade uploaders. In the user profile form only the active semester´s participations can be edited.' %}"></span>
```

Comment:
Some things:
1. The span element should be indented to match the indentation of the button above.
2. The list "managers, reviewers, and grade uploaders" is missing a comma.
3. The apostrophe at "semester's participations" does not seem to be a normal `'` character but something else.